cmd/ingress/main.go

```diff
@@ -92,26 +92,22 @@ func main() {
 		logger.Errorw("failed to create the metrics exporter", zap.Error(err))
 	}
 
-	rmqHelper := rabbit.NewRabbitMQHelper(1, make(chan bool))
+	rmqHelper := rabbit.NewRabbitMQHelper(1, make(chan bool), rabbit.DialWrapper)
 	// Wait for RabbitMQ retry messages
+	defer rmqHelper.CleanupRabbitMQ(env.connection, logger)
 	go func() {
```

Comment:
The connection/channel is initialized async along with the HttpServer that would receive events. Should we be protecting against a NPE at around L200? We don't check that the channel exists before attempting to send the event out.